### PR TITLE
Feature/remove tls context on web server

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: src/main/java/com/iexec/sms/ssl/SslConfig.java [hunk 1]
```diff
@@ -29,19 +30,20 @@
 import java.security.UnrecoverableKeyException;
 import java.security.cert.CertificateException;
 
+@Slf4j
```
Comment (jeremyjams) [inline src/main/java/com/iexec/sms/ssl/SslConfig.java:33]: Enable if SCONE?

File: src/main/resources/application-gramine.yml [hunk 2]
```diff
@@ -21,5 +21,5 @@ tee:
 
   gramine:
     sps:
-      login: ${IEXEC_GRAPHENE_SPS_WEB_LOGIN:admin}
-      password: ${IEXEC_GRAPHENE_SPS_WEB_PASSWORD:admin}
+      login: ${IEXEC_GRAMINE_SPS_WEB_LOGIN:admin}
```
Comment (jeremyjams) [inline src/main/resources/application-gramine.yml:24]: :+1:

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -59,4 +50,12 @@ logging:
 
 springdoc:
   packagesToScan: com.iexec.sms
-  pathsToMatch: /**
\ No newline at end of file
+  pathsToMatch: /**
+
+tee:
```
Comment (jeremyjams) [inline src/main/resources/application.yml:55]: Can we move that block to `application-scone.yml`?

**Review by mcornaton — APPROVED** [review]
Looks great, thanks!

File: src/main/resources/application.yml (hunk 3 above)
Comment (jbern0rd) [reply src/main/resources/application.yml]: See d7fee20

File: src/main/java/com/iexec/sms/ssl/SslConfig.java (hunk 1 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/sms/ssl/SslConfig.java:35]: See a01febc

Comment (jeremyjams) [reply src/main/java/com/iexec/sms/ssl/SslConfig.java:35]: Nice thanks :)

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -1,14 +1,5 @@
...
-  http:
-    enabled: ${IEXEC_SMS_HTTP_ENABLED:true}
-    port: ${IEXEC_SMS_HTTP_PORT:13300}
-  ssl:
-    key-store: ${IEXEC_SMS_SSL_KEYSTORE:./src/main/resources/ssl-keystore-dev.p12} #iexec-core dev certificate for dev
-    key-store-password: ${IEXEC_SMS_SSL_KEYSTORE_PASSWORD:whatever}
-    key-store-type: ${IEXEC_SMS_SSL_KEYSTORE_TYPE:PKCS12}
-    key-alias: ${IEXEC_SMS_SSL_KEYSTORE_ALIAS:iexec-core}
-    client-auth: need
+  port: ${IEXEC_SMS_PORT:13300} # document variable name modification for release
```
Comment (jeremyjams) [inline src/main/resources/application.yml:2]:
```suggestion
  port: ${IEXEC_SMS_PORT:13300} #TODO: document variable name modification for release
```

**Review by jeremyjams — APPROVED** [review]
Thank you for your updates!

File: src/main/resources/application.yml (hunk 4 above)
Comment (jbern0rd) [reply src/main/resources/application.yml]: see ff3b811

**Pull request merged**